chsdi/models/bod.py

```diff
@@ -206,12 +205,16 @@ class GetCap(object):
     datenherr = Column('datenherr', Unicode)
     wms_kontakt_abkuerzung = Column('wms_kontakt_abkuerzung', Unicode)
     wms_kontakt_name = Column('wms_kontakt_name', Unicode)
-    zoomlevel_min = Column('zoomlevel_min', Integer)
-    zoomlevel_max = Column('zoomlevel_max', Integer)
```

Comment:
never ever do that

Reply:
the bod migration script pr has been open for over a month...

Reply:
we can remove from the code but not from db